projects/storefrontapp/src/environments/environment.ccv2.ts

```diff
@@ -0,0 +1,5 @@
+export const environment = {
+  production: true,
+  occBaseUrl:
+    'https://api.c39j2-walkersde1-d4-public.model-t.cc.commerce.ondemand.com',
```

Comment:
not sure if this is the one we want since I thought we're suppose to use the `occ` prefix and I remember the container with `storefront` in the ccv2 environment needs to be used as its where occ endpoints are at

BUT! it looks like it works for api. back. and storefront. 
What I notice is that when pointing to api, it uses the `rest` prefix as the default and is the ycommerceweb addon, while the other two uses `occ` prefix as the default, which is commerceweb extension

Reply:
nvm I'm wrong :) I think 1905 is deployed to the ccv2 instance

Reply:
BUT yeah why api. and not storefront. or backoffice. ?

Reply:
On CCv2, the url prefix of the node that exposes the apis is `api`